src/DynamoPackages/PackageLoader.cs

```diff
@@ -211,6 +211,19 @@ private void ScanPackageDirectories(string root, IPreferences preferences)
             {
                 if (!Directory.Exists(root))
                 {
+                    string extension = null;
+                    if (root != null)
+                    {
+                        extension = Path.GetExtension(root);
+                    }
+
+                    // If the path has a .dll or .ds extension it is a locally imported library
+                    // so do not output an error about the directory
+                    if (extension == ".dll" || extension == ".ds")
```

Comment:
what does this avoid - a log to the console?

Reply:
Yes, as mentioned in the purpose section of the initial PR. There were messages being logged to the console in the case that the "root" was a file instead of a directory. This only occurs in the case that a library was imported using the "File | ImportLibrary..." command. The message said that the directory could not be found, which was because the "directory" was actually a filename and not a directory. The information that the file could not be loaded will have already been logged to the console if needed.